src/main/java/wolox/bootstrap/models/Role.java

```diff
@@ -1,16 +1,10 @@
 package wolox.bootstrap.models;
 
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import javax.persistence.*;
```

Comment:
I would remove wildcards

Reply:
https://intellij-support.jetbrains.com/hc/en-us/community/posts/206203659-Turn-off-Wildcard-imports-

Reply:
Yes! Great idea, to avoid collision and show explicitly what we are importing :) It was replaced by the IntelliJ, I would change that IDE configuration 👍

Reply:
Done but in the next PR: https://github.com/Wolox/java-bootstrap/pull/30